Github Actions fails to deploy documentation

## Issue Type

<!--- Pick one below and delete the rest -->
- Other

## Summary

During CI, actions is configured to install a requirements file which is not existing in the repository

## Error Message

```shell
Run pip install -r requirements.txt
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
##[error]Process completed with exit code 1.

```

--- a/.github/workflows/documentation-build.yml
+++ b/.github/workflows/documentation-build.yml
@@ -13,7 +13,6 @@ jobs:
           python-version: '3.x'
       - name: 'Build mkdocs content to site folder'
         run: |
-            pip install -r requirements.txt
             pip install -r ansible_collections/arista/avd/docs/requirements.txt
             make webdoc
       - name: 'Deploy last version to gh-pages'

